ceph.spec.in

```diff
@@ -438,12 +440,16 @@ Requires:       python%{_python_buildid}-werkzeug
 %if 0%{?suse_version}
 Requires:       python%{_python_buildid}-CherryPy
 Requires:       python%{_python_buildid}-Werkzeug
-Recommends:     python%{_python_buildid}-influxdb
+%endif
+%if 0%{?weak_deps}
 Recommends:	ceph-mgr-dashboard = %{_epoch_prefix}%{version}-%{release}
 Recommends:	ceph-mgr-diskprediction-local = %{_epoch_prefix}%{version}-%{release}
 Recommends:	ceph-mgr-diskprediction-cloud = %{_epoch_prefix}%{version}-%{release}
 Recommends:	ceph-mgr-rook = %{_epoch_prefix}%{version}-%{release}
 Recommends:	ceph-mgr-ssh = %{_epoch_prefix}%{version}-%{release}
+%if 0%{?fedora} || 0%{?suse_version}
+Recommends:     python%{_python_buildid}-influxdb
+%endif
```

Comment:
Why have this conditional inside the `%if 0%{?weak_deps}` conditional block? It's effectively duplicating the outer conditional?

Reply:
i was trying to convince myself that RHEL8 will be using rpm 4.12. but i failed to find any clues what rpm version RHEL8 will be using, but i forgot to remove after re-searching "RHEL8" and "rpm".

will drop this condition.